### PR TITLE
macOS Installer Release Guide

### PR DESCRIPTION
**Pull request by dasmanas**
## Description

This PR enhances the macOS Installer Release Guide.

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've run the unit tests `brew test pyrsia` and everything passes.
- [x] I've made sure my brew is current and updated.

### CODE REVIEW

**Review by fishseabowl — APPROVED** [review]

**Review by betarelease — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -49,4 +49,36 @@ pyrsia 0.2.0
...
+    rect rgb(200, 150, 255)
+    note right of gcb: homebrewrepo/stable/x86_64/<archive_name>
+    note right of gcb: accessible using https://brewrepo.pyrsia.io/stable/x86_64/<archive_name>
+    end
+    end
+    rect rgb(100, 255, 190)
+    user ->> hp: manually update Formula/pyrsia.rb
+    user -->> hp: update archive_name in the url
+    user -->> hp: calculate sha256 and update sha256
+    user -->> hp: update version
```
Comment (ihcomega56) [inline README.md:81]: Are dashed lines opposite?

```suggestion
    hp -->> user: update archive_name in the url
    hp -->> user: calculate sha256 and update sha256
    hp -->> user: update version
```
image

**Review by ihcomega56 — COMMENTED** [review]
One question left.

File: README.md (hunk 1 above)
Comment (dasmanas) [reply README.md:81]: here dashed line are subcomponents of what user needs to update manually. 3 elements need to be updated manually i.e. url, sha256, version.

Comment (ihcomega56) [reply README.md:81]: @dasmanas Ah okay thank you. I thought users should just run `brew update` and `brew upgrade pyrsia`.
Do we have documents which specific commands and procedures are written?

Comment (ihcomega56) [reply README.md:81]: If not, I will add the info.

**Pull request merged**